server/controllers/cohort.js

```diff
@@ -133,27 +141,63 @@ const updateCohortById = (req, resp) => {
         } else {
           cohortActivity = ActivityType.COHORT_EDIT_DESCRIPTION;
         }
+
+        const data = { cohortId, description };
+
+        return socketActions.updateCohort(
+          socketInstance,
+          allCohortsViewClients
+        )(data);
       }
 
       if (name) {
         cohortActivity = ActivityType.COHORT_EDIT_NAME;
+
+        const data = { cohortId, name };
+
+        return socketActions.updateCohort(
+          socketInstance,
+          allCohortsViewClients
+        )(data);
       }
 
       if (isArchived !== undefined) {
-        cohortActivity = isArchived
-          ? ActivityType.COHORT_ARCHIVE
-          : ActivityType.COHORT_RESTORE;
-      }
+        const data = { cohortId };
```

Comment:
you can declare this data once, above all if's as let data;
and ovewrite this data in every single if